modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -282,7 +282,7 @@ const insert = (editor: Editor, evt?: EditorEvent<KeyboardEvent>): void => {
             block.appendChild(clonedNode);
           }
         }
-      } while ((node = node.parentNode) && node !== editableRoot);
+      } while ((node = node.parentNode) && !editableRoot.exists((o) => o === node));
```

Comment:
Option comparison used to be on `Optional`, but it was moved to `Optionals`

I believe this can be changed to `!Optionals.is(editableRoot, node)`